modules/bot-registry/package.json

```diff
@@ -11,12 +11,12 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@connext/utils": "6.5.7",
-    "body-parser": "^1.19.0",
-    "express": "^4.17.1"
+    "@connext/utils": "6.6.3",
```

Comment:
We should make npm-publish update this.

Reply:
Ordinarily it would update `@connext/utils` to the newly published version in all modules, but that change got lost during a merge so I had to do it manually